### PR TITLE
znew: add page

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/znew.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# znew
+
+> Recompress files from `.Z` to `.gz` format.
+> More information: <https://www.gnu.org/software/gzip/manual/gzip.html>.
+
+- Recompress a file from `.Z` to `.gz` format:
+
+`znew {{path/to/file1.Z}}`
+
+- Recompress multiple files and display the achieved size reduction per file:
```
Comment (kbdharun) [inline pages/common/znew.md:10]:
```suggestion
- Recompress multiple files and display the achieved size reduction % per file:
```

File: pages/common/znew.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# znew
+
+> Recompress files from `.Z` to `.gz` format.
+> More information: <https://www.gnu.org/software/gzip/manual/gzip.html>.
```
Comment (kbdharun) [inline pages/common/znew.md:4]:
```suggestion
> More information: <https://manned.org/znew>.
```

File: pages/common/znew.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Recompress a file from `.Z` to `.gz` format:
+
+`znew {{path/to/file1.Z}}`
+
+- Recompress multiple files and display the achieved size reduction per file:
+
+`znew -v {{path/to/file1.Z}} {{path/to/file2.Z}} {{path/to/file3.Z}}`
+
+- Recompress a file using the optimal (and slowest) compression method:
```
Comment (kbdharun) [inline pages/common/znew.md:14]:
```suggestion
- Recompress a file using the slowest compression method (for optimal compression):
```

File: pages/common/znew.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`znew -9 {{path/to/file1.Z}}`
+
+- Recompress a file, keeping the `.Z` file if it is smaller than the `.gz` file:
```
Comment (kbdharun) [inline pages/common/znew.md:18]:
```suggestion
- Recompress a file, [K]eeping the `.Z` file if it is smaller than the `.gz` file:
```

> Tip: Try to use mnemonics whenever possible.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**